src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.ts

```diff
@@ -182,6 +186,9 @@ export class OsdFormComponent extends CdForm implements OnInit {
       this.enableFeatures();
     }
     this.driveGroup.setDeviceSelection(event.type, event.filters);
+    if (this.clusterCreation) {
```

Comment:
Even when the disk was created in the newly-added host:
```
# ceph orch device ls
Hostname                  Path      Type  Serial  Size   Health   Ident  Fault  Available  
ceph-node-00.cephlab.com  /dev/vdb  hdd           5368M  Unknown  N/A    N/A    Yes        
ceph-node-01.cephlab.com  /dev/vdb  hdd           5368M  Unknown  N/A    N/A    Yes
```

In OSD creation modal the aforementioned disk was not available for selection,
so we need to implement data refresh in order to reflect all available disks.

Reply:
My thoughts on this: 

With the cephadm agent, we're going to have the ability to implement reasonably fast drive refresh. By this point, we're not going to need to implement a refresh notification. I think this improvement will land within the next two months. 

Till then, it's going to take an awful long time to refresh drives. you can trigger a refresh using `ceph orch device ls <hostname> --refresh`, but that's not synchronous.

Reply:
@sebastian-philipp is there any way from cephadm to know that whether the added host will have a disk or not? Rather than waiting for the `ceph orch device ls` output?